boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,171 @@ currently supports the following hardware features:
 
 Building
 ========
 
-Applications for the ``it8xxx2_evb`` board configuration can be built as usual
-(see :`Zephyr Getting Started Guide`_):
+#. Build :ref:`hello_world` application as you would normally do
+   (see :`Zephyr Getting Started Guide`_):.
+
+   .. zephyr-app-commands::
+      :board: it8xxx2_evb
+      :zephyr-app: samples/hello_world
+      :goals: build
 
-.. zephyr-app-commands::
-   :board: it8xxx2_evb
-   :goals: build
+#. The file :file:`zephyr.bin` will be created by west.
 
 Flashing
 ========
 
-In order to upload the application to the device,
-you'll need our winflash tool and download board.
-You can get the winflash tool at here:`Download Link`_
-Use the winflash.exe tool to program a zephyr application
+Windows
+--------
+
+Use the winflash tool to program a zephyr application
 to the it8xxx2 board flash.
 
+#. Open winflash tool and make sure the order you open the switch is right.
+   Fisrt, turn on the Download board switch.
+   Second, turn on the it8xxx2_evb board switch.
+   Then, configure your winflash tool like below.
+
+    .. figure:: WinFlashTool_P2.png
+          :width: 600px
+          :align: center
+
+    .. figure:: WinFlashTool_P4.png
+          :width: 600px
+          :align: center
+
+#. Using winflash tool flash zephyr.bin into your ITE board.
+   First, click ``Load`` button and select your zephyr.bin file.
+   Second, click ``run`` to flash the iamge into board.
+
+    .. figure:: WinFlashTool_P3.png
+          :width: 600px
+          :align: center
+
+#. At this point, you have flashed your image into ITE board and
+   it will work if you turn on ITE board. You can use a terminal program
+   to verify flashing worked correctly.
+
+   For example, open device manager to find the USB Serial Port(COM4) and use your
+   terminal program to connect it(Speed: 115200).
+
+    .. figure:: WinFlashTool_P1.png
+          :width: 600px
+          :align: center
+
+#. Turn on the it8xxx2_evb board switch, you should see ``"Hello World! it8xxx2_evb"``
+   sent by the board. If you don't see this message, press the Reset button and the
+   message should appear.
+
+Ubuntu
+--------
+
+#. Run your favorite terminal program to listen for output.
+   Under Linux the terminal should be :code:`/dev/ttyUSB0`. Do not close it.
+
+   For example:
+
+   .. code-block:: console
+
+      $ minicom -D /dev/ttyUSB0 -b 115200
+
+#. Open a second terminal window and use ``west`` to flash your board.
+   Split first and second terminal windows to view both of them.
+
+   .. code-block:: console
+
+      $ west flash
+
+#. You should see ``"Hello World! it8xxx2_evb"`` in the first terminal window.
+   If you don't see this message, press the Reset button and the message should appear.
+
 Debugging
 =========
 
 Supporting uart debug, currently.
 
+Troubleshooting
+===============
+
+#. If the flash tool reports a failure, re-plug the 8390 Download board or
+   power on the ec and try again.
```

Comment:
"power on the ec"?  Do you mean power cycle the it8xxx2_evb board?

Reply:
Yes, "ec" is confusing. I have fixed it.